addons/event/models/event_tag.py

```diff
@@ -15,10 +15,11 @@ class EventTagCategory(models.Model):
     sequence = fields.Integer('Sequence', default=0)
     tag_ids = fields.One2many('event.tag', 'category_id', string="Tags")
 
+
 class EventTag(models.Model):
     _name = "event.tag"
     _description = "Event Tag"
-    _order = "sequence"
+    _order = "category_id, sequence"
```

Comment:
What about storing category_id.sequence on event.tag model, then using "_order = category_sequence, sequence" ? That way we are sure order is logical based on category sequences.